### PR TITLE
Fix or remove broken links

### PR DESCRIPTION
**Pull request by DoryanR**
Fixed broken links and invalid redirects on the whole wiki. Thanks to [clayton's tool](https://github.com/cl8n/osu-wiki-bin) (cl8n) which saved me a lot of time on this task.

On this PR, you will find the fix of all the links and redirections that were marked as invalid, that did not redirect to anything, to make it short.

I did a check of all the articles, even the ones with the `outdated: true` tag, because I thought that even these articles deserve a special attention for this case, because we don't know when they will be updated, so we might as well take care of them too.
This is what justifies the number of files concerned by this PR.

Despite the efficiency of this tool, I noticed that some errors were not justified (link or redirections correct, but yet noted as invalid). So I determined that these were false positives.
As a matter of conscience, I leave a gist with the last errors found [here](https://gist.github.com/DoryanR/9ddc34ee81a889b3f339f6c461a1ea96).

Thanks in advance for the reviews.

PS: in addition to the links, some improvements have been made to the articles, either because of CI errors, or because they were found during the reviews.

### CODE REVIEW

**Comment by DoryanR** [comment]
Ready for reviews

File: wiki/Beatmap/fr.md
Comment (TicClick) [inline wiki/Beatmap/fr.md:29]: in addition, links in other regular articles must be absolute (example: `/wiki/Beatmap/Beatmap_host`), don't make them relative if they aren't that already
```suggestion
Les auteurs de beatmaps peuvent [publier](/wiki/Submission) leur création dans la [liste des beatmaps](https://osu.ppy.sh/beatmapsets). Bien que chaque beatmap est attribuée à [une seule personne](/wiki/Beatmap/Beatmap_host), il s'agit souvent d'un effort commun: certaines difficultés peuvent être créées par d'autres personnes qui travaillent [en collaboration](/wiki/Beatmap/Beatmap_collaborations) ou [séparément](/wiki/Beatmap/Guest_difficulty).
```

File: wiki/Beatmap/pt-br.md
Comment (TicClick) [inline wiki/Beatmap/pt-br.md:17]:
```suggestion
- [*storyboard*](/wiki/Storyboard) com gráficos e efeitos especiais, servindo como história de fundo ou tema para a música (opcional).
```

File: wiki/Beatmap/ru.md
Comment (TicClick) [inline wiki/Beatmap/ru.md:55]: no need to capitalize it

File: wiki/Beatmap_Editor/fr.md
Comment (TicClick) [inline wiki/Beatmap_Editor/fr.md:11]: this is not a table of contents, remove the first/last items

File: wiki/Beatmap_Editor/en.md
Comment (TicClick) [inline wiki/Beatmap_Editor/en.md:41]: why remove a perfectly valid item list?

File: wiki/Beatmap_Editor/de.md
Comment (TicClick) [inline wiki/Beatmap_Editor/de.md:22]: why remove it?

File: wiki/Beatmap_Editor/fr.md
Comment (TicClick) [inline wiki/Beatmap_Editor/fr.md:39]: please don't remove it, fix the link instead

**Review by TicClick — CHANGES_REQUESTED** [review]
I've checked like 20% but have to ask you to review your own changes, there are some odd ones

File: wiki/Beatmap/fr.md
Comment (DoryanR) [reply wiki/Beatmap/fr.md:29]: Since it was in the `Beatmap` root, I thought it was feasible, as I had already seen on other articles. This will be modified

File: wiki/Beatmap_Editor/fr.md
Comment (DoryanR) [reply wiki/Beatmap_Editor/fr.md:11]: Same thing for `Banchobot`, it will be corrected

File: wiki/Beatmap_Editor/fr.md
Comment (DoryanR) [reply wiki/Beatmap_Editor/fr.md:39]: It will be done

File: news/2019-04-11-beatmap-spotlights-winter-2019.md
Comment (TicClick) [reply news/2019-04-11-beatmap-spotlights-winter-2019.md:9]: when you push an article or a newspost, and then change the image without renaming it, the change doesn't get displayed, as the image is cached by the website up to 60 days. to force refresh it, people add a dummy query string or rename the image.

if you see a link with `?1` or something that is more than sixty days old, it should be safe to remove the `?1` part, it does absolutely nothing useful other than make the website think it's a different image

File: wiki/Article_styling_criteria/Formatting/en.md
Comment (TicClick) [reply wiki/Article_styling_criteria/Formatting/en.md:624]: there is no redirect, is there?

Comment (DoryanR) [reply wiki/Article_styling_criteria/Formatting/en.md:624]: Hmm, I have the impression that I made a small mistake in the end, I had to remove the two redirects instead of one, because the flags update had added a link at the bottom of the page, and it was him who created the error, not that one. This will be corrected on the next commits.

File: news/2019-04-11-beatmap-spotlights-winter-2019.md
Comment (DoryanR) [reply news/2019-04-11-beatmap-spotlights-winter-2019.md:9]: Thanks for these explanations, it is more clear on this parameter.

**Comment by peppy** [comment]
> I also took the opportunity to update the flags (via the `update-flags` command), it doesn't necessarily have anything to do with the purpose of the PR, but since it's sort of about links, I thought it was a good idea.

The review touches so many lines this is impossible to check. Please do this in a separate PR.

> You are free to rebase, I still leave the traces for a simpler review, which may be long.

I don't understand how many individual commits all with the same commit message help with review.

**Comment by Krystool** [comment]
Yeah, there are some odd ones. I probably would've separated the images from the actual pages links in two distinct PRs, but have a checklist

- [ ] review images
- [ ] review links
- [ ] review others
- [ ] all good

File: wiki/Beatmap_Spotlights/id.md
Comment (DoryanR) [inline wiki/Beatmap_Spotlights/id.md:196]:
```suggestion
[flag_HU]: /wiki/shared/flag/HU.gif "Hungaria"
```

**Comment by DoryanR** [comment]
*(from peppy)* :
> I don't understand how many individual commits all with the same commit message help with review.

It seemed clearer to me than having a single commit with everything on it.
By having the commits represent the sessions where I was working on the changes, it was easier for me than doing it article by article.

**Comment by DoryanR** [comment]
On my side all the modifications were made, I proceeded to a re-reading and all seems to me good, I am now waiting for a review to confirm my statements.

File: wiki/Help_Centre/Client/ru.md
Comment (TicClick) [inline wiki/Help_Centre/Client/ru.md:274]:
```suggestion
В некоторых ситуациях сброс статуса можно исправить, открыв перед началом игры таблицу мировых рекордов. Если статус сбросился у многих карт (или вообще у всех), попробуйте [удалить базу данных карт](#missing-beatmaps). После этого у карт должны вскоре появиться правильные статусы.
```

File: wiki/Interface/de.md
Comment (TicClick) [inline wiki/Interface/de.md:66]:
```suggestion
Dieser Bereich zeigt ihnen **Informationen zu der Beatmap und der ausgewählten Schwierigkeitsstufe an.** Standardmäßig wird die Beatmap angezeigt, welche im osu! Jukebox abgespielt wurde. Die BPM-Anzeige zeigt den größten und niedrigsten Wert der Beatmap an. "Objekte" gibt die vollständige Anzahl von [Hit Objekten](/wiki/Hit_Object) in den Beatmap wieder. Ein ![](/wiki/shared/icon/heart.gif)Herz Symbol an der oberen linken Ecke kennzeichnet die Beatmap als Ranked, während ein ![](/wiki/shared/icon/flame.gif) Flamen Symbol die Beatmap als Approved kennzeichnet. Der Unicode-Titel wird benutzt, wenn in den Einstellungen die Option ["Metadaten in Originalsprache anzeigen"](/wiki/Options) aktiviert ist und wenn auch Unicodes im Titel/Artist der Beatmap vorhanden sind.
```

File: wiki/Kiai_time/de.md
Comment (TicClick) [inline wiki/Kiai_time/de.md:7]:
```suggestion
**Kiai Time** wird vom [Mapper](/wiki/Beatmapping) in der [Beatmap](/wiki/Beatmap) zwischen 2 [Timingsektionen](/wiki/Beatmap_Editor/Timing) definiert. Sie werden es durch die Sternenfontänen und an dem Geblinke auf Ihrem Bildschirm erkennen, danach werden alle [Hit Objekte](/wiki/Hit_Object) im Rhythmus zum Hauptbeat anfangen zu blinken und Sterne werden aus Ihrem Cursor erscheinen, wenn Sie einen [Hit Circle](/wiki/Hit_object/Hit_circle) anklicken oder von Ihrem Cursor runterfallen, wenn Sie die Taste bei einem [Slider](/wiki/Hit_object/Slider) oder einem [Spinner](/wiki/Hit_object/Spinner) durchgehend gedrückt halten.
```

**Review by TicClick — CHANGES_REQUESTED** [review]
checked 419 files so far (up to `People/*`)

**Comment by peppy** [comment]
> I was going to try to review this one but it's doing a lot more than fixing links... can the other changes be summarized in the first post or title please
> 
> (ideally not bundled into one PR in the first place, but it's already been reviewed so w/e)

Already brought this up. I can't review this one because it breaks my browser.

**Comment by DoryanR** [comment]
I have, indeed, took a long time to apply everything manually, and I checked every proposal that the tool proposed, to be sure that the link was correct.
It really bothers me to see that the review is impossible to do because of the number of files, while I spent a lot of time on it....

**Comment by TicClick** [comment]
yeah, and the changes are, well, still manual. sorry!

**Review by TicClick — DISMISSED** [review]

**Review by TicClick — APPROVED** [review]
@peppy could you please give it a second review attempt now that it's reduced in size and cleaned up (and checked by me and clayton)? would it help if changes requiring your approval (news) were sent for review separately?

**Comment by DoryanR** [comment]
What does it become? I thought it had been merged by now.

I'm afraid it's going to be hard to keep the PR up to date, with the number of PRs that have been merged in the meantime.

**Comment by peppy** [comment]
@TicClick i'll leave you with responsibility to merge this. i do not have the time or willpower to try and check through it, but trust your judgement.

**Review by peppy — APPROVED** [review]

**Review by TicClick — APPROVED** [review]

**Comment by TicClick** [comment]
instant merge is unavailable to me, apparently because CI checks are failing. while I can't figure out how to fix that specific issue (a list wrapped in a quote), and #6394 didn't help, I suggest we sweep it under the rug for now.

that, unfortunately, leaves merge to you and, maybe, Ephemeral, but I gave the PR a quick look again and it looks good to me.

**Comment by DoryanR** [comment]
I had already fixed this error when I translated the article, but I forgot to do this for the English version. I will make a PR to resolve this problem.

**Comment by DoryanR** [comment]
🎉

**Pull request merged**
